### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by dillfrescott** [AlexBuz/llama-zip#7]
If i compress a large text file with 40% overlap, do I need to specify the correct same overlap during decompression for it to properly decompress or no?

**Comment by AlexBuz** [comment]
Yes, as otherwise the context will be different during decompression than it was during compression, which will affect the model's logits and possibly result in different tokens being chosen.

**Comment by jreoka** [comment]
Okay got it!
